scripts/setup/create_multinode.sh

```diff
@@ -111,20 +111,11 @@ function setup_master() {
         LOGIN_TOKEN="$(server_exec "$MASTER_NODE" 'tmux show-buffer -b token | grep -B 3 "All nodes need to be joined"')"
         echo "$LOGIN_TOKEN"
     done
-    # cut of last line
-    LOGIN_TOKEN=${LOGIN_TOKEN%[$'\t\r\n']*}
-    # remove the \
-    LOGIN_TOKEN=${LOGIN_TOKEN/\\/}
-    # remove all remaining tabs, line ends and returns
-    LOGIN_TOKEN=${LOGIN_TOKEN//[$'\t\r\n']}
-}
-
-function setup_loader() {
-    echo "Setting up loader/monitoring node: $1"
 
-    server_exec "$1" 'wget -q https://go.dev/dl/go1.20.5.linux-amd64.tar.gz >/dev/null'
-    server_exec "$1" 'sudo rm -rf /usr/local/go && sudo tar -C /usr/local/ -xzf go1.20.5.linux-amd64.tar.gz >/dev/null'
-    server_exec "$1" 'echo "export PATH=$PATH:/usr/local/go/bin" >> .profile'
+    # Get the join token from k8s.
+    pushd $HOME/vhive/scripts > /dev/null
```

Comment:
This is wrong, the scripts write that on master node, not local machine. We should execute it through the ssh, not this way.